tests/Fixtures/ParamsFixture.php

```diff
@@ -353,9 +354,9 @@ public static function getEventsApiSignatureForVersion(string $version): string
     {
         switch ($version) {
             case '01':
-                return '20739eed410d54a135e8cb3745628834886ab315bfc01693ce9acc0d14dc98bf';
+                return 'fd236f3bc3e14bb291a222637fe86d71c3d4393ed7c51353a2f9a14b356fd605';
             case '02':
-                return '$02$5c3160dbb9ab4d01774b5c2fc3b01a35ce4f9709c84571c27dfe333d1ca9d349';
+                return '$02$e319a1c9c3e0118d1e81188a650b054a765e072a2d90e4b422ecca2ae9589449';
```

Comment:
It would be worth propagating those changes to the test-suite of the other SDKs. They should be using the same payload/signature pairs.